srai/plotting/plotly_wrapper.py

```diff
@@ -19,6 +20,69 @@
 import_optional_dependencies(dependency_group="plotting", modules=["plotly"])
 
 
+def plot_numeric_data(
+    regions_gdf: gpd.GeoDataFrame,
+    data_column: str,
+    data_df: Optional[Union[gpd.GeoDataFrame, pd.DataFrame]] = None,
+    colormap: Union[str, List[str]] = px.colors.sequential.Sunsetdark,
```

Comment:
This colormap, which is the default, throws an error. I suppose that this is because this one is in RGB format so it is not a `List[str]` but `List[Tuple]`

```
ValueError: 
    Invalid value of type 'builtins.list' received for the 'colorscale' property of make_figure
```